backend/corpora/api_server/app.py

```diff
@@ -19,8 +19,14 @@ def create_flask_app():
     connexion_app = connexion.FlaskApp(f"{APP_NAME}-{DEPLOYMENT_STAGE}", specification_dir="backend/config")
     # From https://github.com/zalando/connexion/issues/346
     connexion_app.app.url_map.strict_slashes = False
-    swagger_spec_path = f"{APP_NAME}.yml"
-    connexion_app.add_api(swagger_spec_path, validate_responses=True)
+    dataportal_api = super(connexion.FlaskApp, connexion_app).add_api(
+        f"{APP_NAME}.yml", validate_responses=True, options={"swagger_ui": True}
+    )
+    curator_api = super(connexion.FlaskApp, connexion_app).add_api(
+        "curator-api.yml", validate_responses=True, options={"swagger_ui": True}
```

Comment:
I'm noticing that we didn't specify `swagger_ui: True` previously. It existed by default though, right? But we aren't publicly advertising the url...? 🤔

Reply:
Yes it wasn't advertised. I'm being explicit here for simplicity.